packages/clerk-js/src/core/fapiClient.ts

```diff
@@ -32,8 +31,21 @@ export type FapiRequestCallback<T> = (
   response?: FapiResponse<T>,
 ) => Promise<unknown | false> | unknown | false;
 
-const camelToSnakeEncoder: qs.IStringifyOptions['encoder'] = (str, defaultEncoder, _, type) => {
-  return type === 'key' ? camelToSnake(str) : defaultEncoder(str);
+const customDefaultEncoder = (value: string) => {
+  return encodeURIComponent(value).replace(/[!'()*]/g, c => '%' + c.charCodeAt(0).toString(16).toUpperCase());
+};
+
+const camelToSnakeEncoder = (str: string, type: 'key' | 'value') => {
+  return type === 'key' ? camelToSnake(str) : customDefaultEncoder(str);
+};
+const stringifyBody = obj => {
+  return Object.keys(obj)
+    .map(key => {
+      const encodedKey = camelToSnakeEncoder(key, 'key');
+      const encodedValue = obj[key] && camelToSnakeEncoder(obj[key], 'value');
+      return encodedValue && `${encodedKey}=${encodedValue}`;
+    })
+    .join('&');
```

Comment:
The map + join operation here makes sense,however, I think we can replace this snippet using the native [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) class

Reply:
Also, is this something we could use the commented-out `stringifyQueryParams` for?